lighthouse-core/test/audits/service-worker-test.js

```diff
@@ -193,7 +197,9 @@ describe('Offline: service worker audit', () => {
     const manifest = {start_url: finalUrl};
 
     const output = ServiceWorker.audit(createArtifacts(swOpts, finalUrl, manifest));
-    assert.deepStrictEqual(output, {score: 1});
+    assert.deepStrictEqual(output.score, 1);
+    assert.deepStrictEqual(output.details.items[0].scopeURL, 'https://example.com/project/');
+    assert.deepStrictEqual(output.details.items[0].scriptURL, 'https://example.com/project/sw.js');
```

Comment:
this is satisfying. and exactly what the test should have been asserting beforehand anyway. without this assertion (`/project/sw` is the controlling sw),  the test was kinda useless.